### ISSUE TITLE
Ensure the app is accessible

### ISSUE THREAD

**Issue by irian-codes** [irian-codes/chat-with-manuals#73]
### Need
An accessibility focused code review should be done to ensure it's accessible. Check [this guide](https://martijnhols.nl/blog/accessibility-essentials-every-front-end-developer-should-know). But when you tackle the aria elements, read [this](https://lucide.dev/guide/advanced/accessibility#a-note-on-aria-label) before.
